change-beta/@azure-communication-react-60233904-ae91-42c7-9b22-9394d4780adb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Should this not be prerelease? I know its a bug fix but it is for a beta feature. @JamesBurnside maybe you can correct me on this?

Reply:
I also believe this should be prerelease as well

Reply:
This PR has two parts: one prerelease and one patch
Prerelease for the fix for the domainpermissions modal
Patch for the fix to the general error bar strings
So I'd keep patch here